doc/Contribution.md

```diff
@@ -0,0 +1,106 @@
+# Contribution Guidelines
+==================
+## Overview
+Being an Open Source project, everyone can contribute, provided that it respect the following points:
```

Comment:
I would suggest using `you` instead of `it`, unless you are referring to me, of course... :p